rest/data/perf.go

```diff
@@ -233,6 +236,65 @@ func (dbc *DBConnector) FindPerformanceResultWithChildren(ctx context.Context, i
 	return apiResults, nil
 }
 
+// ScheduleSignalProcessingRecalculateJobs schedules signal processing recalculation jobs for
+// each project/version/task/test combination
+func (dbc *DBConnector) ScheduleSignalProcessingRecalculateJobs(ctx context.Context) error {
+	db := dbc.env.GetDB()
+	pipeline := bson.A{
+		bson.M{
+			"$match": bson.M{
+				"info.order": bson.M{
+					"$exists": true,
+				},
+				"info.mainline":  true,
+				"info.project":   bson.M{"$exists": true},
+				"info.variant":   bson.M{"$exists": true},
+				"info.task_name": bson.M{"$exists": true},
+				"info.test_name": bson.M{"$exists": true},
+			},
+		},
+		bson.M{
+			"$group": bson.M{
+				"_id": bson.M{
+					"task":    "$info.task_name",
+					"variant": "$info.variant",
+					"project": "$info.project",
+					"test":    "$info.test_name",
+				},
+			},
+		},
+	}
+	cur, err := db.Collection("perf_results").Aggregate(ctx, pipeline)
+	defer cur.Close(ctx)
+	if err != nil {
+		return gimlet.ErrorResponse{StatusCode: http.StatusInternalServerError, Message: fmt.Sprint("Failed to aggregate recalculation metrics")}
+	}
+	for cur.Next(ctx) {
+		var result units.MetricGrouping
+		err := cur.Decode(&result)
+		if err != nil {
+			grip.Error(map[string]interface{}{
+				"message": "Unable to to decode aggregation result",
```

Comment:
I'd use message.WrapError, rather than a map

Reply:
Doesn't that also also accept `interface{}`? How should I be instantiating this `interface{}` if not with a map? I'm guessing there is a much simpler way that has eluded me 😅

Reply:
bump

Reply:
I think this has been done.

Reply:
that doesn't seem to be reflected in the latest patch.